src/explorer.api/Models/ExploreResult.cs

```diff
@@ -31,14 +31,19 @@ public ExploreResult(Guid explorationId, Exploration exploration, VersionInfo ve
                 new ColumnMetricsCollection(
                     ce.Column,
                     ce.PublishedMetrics.Select(m => new Metric(m.Name, m.Metric))));
-            VersionInfo = versionInfo;
         }
 
         public Guid Id { get; }
 
         public ExplorationStatus Status { get; }
```

Comment:
It might be useful to add a separate `Status` field for each column exploration.

Reply:
That's a good idea, I'll log an issue for this.

Reply:
(#199)